Doppler.Sap.Job/DopplerSapJob.cs

```diff
@@ -2,47 +2,54 @@
 using System.Threading.Tasks;
 using CrossCutting;
 using Doppler.Sap.Job.Service.DopplerCurrencyService;
-using Doppler.Sap.Job.Service.Logger;
+using Doppler.Sap.Job.Service.DopplerSapService;
 using Hangfire;
+using Microsoft.Extensions.Logging;
 
 namespace Doppler.Sap.Job.Service
 {
     public class DopplerSapJob : IRecurringJob
     {
-        private readonly ILoggerAdapter<DopplerSapJob> _logger;
+        private readonly ILogger<DopplerSapJob> _logger;
         public string Identifier { get; }
         public string IntervalCronExpression { get; }
 
         private readonly IDopplerCurrencyService _dopplerCurrencyService;
+        private readonly IDopplerSapService _dopplerSapService;
 
         public DopplerSapJob(
-            ILoggerAdapter<DopplerSapJob> logger,
+            ILogger<DopplerSapJob> logger,
             string intervalCronExpression,
             string identifier,
-            IDopplerCurrencyService dopplerCurrencyService)
+            IDopplerCurrencyService dopplerCurrencyService,
+            IDopplerSapService dopplerSapService)
         {
             _logger = logger;
             IntervalCronExpression = intervalCronExpression;
             Identifier = identifier;
             _dopplerCurrencyService = dopplerCurrencyService;
+            _dopplerSapService = dopplerSapService;
         }
 
         [AutomaticRetry(Attempts = 0)]
-        public void Run()
+        public object Run()
         {
-            RunAsync().GetAwaiter().GetResult();
+            var a = RunAsync().GetAwaiter().GetResult();
+            return a;
         }
 
-        private async Task RunAsync()
+        private async Task<object> RunAsync()
```

Comment:
What is the meaning of this returned object?

Reply:
The meaning is that RunAsync() method return the object to displays in Hangfire UI the object created to send to DopplerSap
![X7bwj](https://user-images.githubusercontent.com/6796523/77947096-c510d980-7299-11ea-8f9f-3ed7ab352a53.png)


See [link](https://stackoverflow.com/questions/57748867/how-to-return-a-return-value-from-hangfire)

Reply:
👍 

It is ugly, but it is Hangfire interface, so, I am fine. Thanks!

Reply:
Maybe is possible to return an `IList<CurrencyResponse>`, let me check

Reply:
Nah, Object is fine